node/src/service.rs

```diff
@@ -441,15 +444,28 @@ async fn start_node_impl(
             para_id,
             relay_chain_interface: relay_chain_interface.clone(),
             net_config,
+            sybil_resistance_level: CollatorSybilResistance::Resistant,
```

Comment:
re-check

Reply:
Apparently this is for consensus mechanisms where there is no sybil-resistance, unlike Aura (our case) or nimbus